app/models/manageiq/providers/vmware/infra_manager/refresh_parser/filter.rb

```diff
@@ -92,6 +94,46 @@ def vm_inv_by_host_inv(host_inv)
       vm_inv
     end
 
+    def dvswitch_and_dvportgroup_inv_by_host_mor(host_mor)
+      dvswitch_inv    = {}
+      dvportgroup_inv = {}
+
+      dvswitches   = @vc_data[:dvswitch] || {}
+      dvportgroups = @vc_data[:dvportgroup] || {}
+
+      dvswitches.each do |dvs_mor, dvs_data|
+        summary = dvs_data["summary"]
+        next if summary.nil?
+
+        dvs_hosts = summary["host"].to_a + summary["hostMember"].to_a
```

Comment:
The first part (```summary["host"]```) may not get the hosts. Please see if [this](https://github.com/jameswnl/manageiq/blob/1332cc198acea011fd4d78b7e430622e634b32a6/app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb#L84-L84) can be used here.

The 2nd part (```summary["hostMember"]```) should be fine.